pages/1.13/deploying-jobs/quickstart/index.md

```diff
@@ -54,51 +49,9 @@ You can manage the most common job-related activity interactively through the DC
-
-    ![Create JOB UI](/1.13/img/New-Job-General.png)
-
-    You can configure the job using the fields displayed or click **JSON Editor** to edit the JSON directly.
-
-1. Click **General** to edit the most basic job settings, such as the kob identifier, CPU, memory, and disk requirements.
-
-    * **ID** - The ID of your job.
-    * **Description** - A description of your job.
-    * **CPUs** - The amount of CPU your job requires.
-    * **Mem** - The amount of memory, in MB, your job requires.
-    * **Disk space** - The amount of disk space, in MB, your job requires.
-    * **Command** - The command your job will execute. Leave this blank if you will use a Docker image.
-
-1. Click **Schedule**, then click **Run on a Schedule** to specify a schedule for when the job runs.
-
-    After you select **Run on a Schedule**, you can use the following settings to define the schedule:
-    - Select **Cron Schedule**  to specify the schedule in `cron` format. Use [this crontab generator](http://crontab.guru) for help.
-    - Select **Time Zone**  to enter the time zone in [TZ format](http://www.timezoneconverter.com/cgi-bin/zonehelp), for example, America/New_York.
-    - Select **Starting Deadline** to set the time, in seconds, to start the job if it misses its scheduled time for any reason. Missed job executions are counted as failed jobs.
-
-1. Click **Docker Container** to specifiy the Docker image for the new job, if you are using one.
-
-    Optionally, you can select **privileged** to run the specified Docker image in privileged mode.
-
-1. Click **Docker Parameters** to specify any additional Docker runtime parameters for the new job, if applicable.
-
-    You can add multiple parameter names and corresponding values.
-
-1. Click **Labels** to specify a **Label Name** and **Label Value** you want to attach as metadata to the new job. You can then use the job label to filter or expose information for labeled jobs. For more information about using labels, see [Labeling tasks and jobs](/1.13/tutorials/task-labels/).
-
-1. Verify that you have added the new job by clicking **Jobs**.
-
-    ![Jobs List](/1.13/img/GUI-Job-List.png/)
-
-## Add a job to a job group
-You can add a job to a an existing job group or create a new job group when you create the job. Use dots in your job ID to nest the job in a group. For example, if you add a job using the job ID `marketing.myjob`, the new `myjob` is created in the `marketing` job group. In DC/OS Enterprise, you can use job groups to implement fine-grained user access. For more information about controlling access to jobs through job groups, see [Granting access to jobs](/1.13/deploying-jobs/job-groups/).
->>>>>>> staging
 
 1. Click **Schedule**, then click **Enable Schedule** to specify a schedule for when the job runs.
 
-<<<<<<< HEAD
     After you select **Enable Schedule**, you can use the following settings to define the schedule:
```

Comment:
Enable schedule is not required to be selected in order to fill out the rest of the schedule fields

Reply:
So, is the point of this option that you can set up a schedule, then enable or disable it ad hoc after creating the job?
Also what is the unit of measure for the Keep Trying setting? I get 0 means keep trying forever/never stop trying but what does 5 mean--5 retry attempts, 5 minutes, 5 seconds, etc?

thanks-LG

Reply:
Yes exactly - you can add a schedule without necessarily enabling it. For Keep Trying Time, the number represents number of seconds to wait before trying to restart the job.